Sync/BookmarkPayload.swift

```diff
@@ -164,7 +165,7 @@ open class SeparatorPayload: BookmarkBasePayload {
         if !super.isValid() {
             return false
         }
-        if !self["pos"].isInt {
+        if self["pos"].int == nil {
```

Comment:
Nit: I saw below that you defined helpers for this in the JSON extension. Could you use those here?